From d3ef9a7315c1742b1d86c32444f5f07e50fa5fbf Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Micha=C5=82=20G=C3=B3rny?= <mgorny@gentoo.org>
Date: Fri, 10 Nov 2023 21:15:46 +0100
Subject: [PATCH] Fix `pyproject.toml` not to install tests as top-level
 package

Fix the `pyproject.toml` configuration not to install e.g.:

    /usr/lib/python3.11/site-packages/tests
---
 pyproject.toml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/pyproject.toml b/pyproject.toml
index aa8b90a..2fdfefc 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -52,7 +52,7 @@ path = 'flask_frozen/__init__.py'
 exclude = ['.*']
 
 [tool.hatch.build.targets.wheel]
-exclude = ['docs']
+exclude = ['docs', 'tests']
 
 [tool.hatch.envs.doc]
 features = ['doc']
